scripts/landing.js

```diff
@@ -9,4 +10,9 @@ var animatePoints = function() {
   for (var i = 0; i < points.length; i++) {
     revealPoint(i);
   }
-};
\ No newline at end of file
+};
+window.onload = function() {
+  window.addEventListener('scroll', function(event) {
```

Comment:
format here is addEventListener('type', function(event){callback}, useCapture); useCapture is optional and we're not using it here.